config/traefik/examples/http-01/docker-compose.override.yml

```diff
@@ -0,0 +1,93 @@
+version: "3.7"
+
+services:
+  traefik:
+    # The official v2 Traefik docker image
+    image: traefik:latest
+    container_name: traefik
+    restart: always
+    env_file: env
+    ports:
+      - "80:80"
+      - "443:443"
+      - "8080:8080"
+    networks:
+      - "invoiceninja"
+    volumes:
+      # So that Traefik can listen to the Docker events
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      # Dynamic configuration files
+      - "./config/traefik/config:/conf:ro"
+      # Enable Access Log
+      - "./config/traefik/logs/:/var/log/www/"
+      # LetsEncrypt Configuration Storage
+      - "./config/traefik/ssl-files:/ssl"
+    command:
+      # Send usage statistics (or not)
+      - "--global.sendAnonymousUsage=false"
+      # By default, the level is set to ERROR. Alternative logging levels are 
+      # DEBUG, PANIC, FATAL, ERROR, WARN, and INFO.
+      - "--log.level=WARN"
+      # Enable Access Log
+      - "--accesslog.filepath=/var/log/www/access.log"
+      # Enable Dashboard
+      - "--api.insecure=false"
+      - "--api.dashboard=true"
+      - "--api.debug=true"
+      # We are using Docker
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      # (Optional) Set default hostname if not given explicitly
+      - "--providers.docker.defaultRule=Host(`${APP_URL_DOMAIN}`)"
+      # Listen on port 80 (http)
+      - "--entrypoints.web.address=:80"
+      # Listen on port 443 (https)
+      - "--entrypoints.websecure.address=:443"
+      # Listen on port 8080 (traefik Dashboard)
+      - "--entrypoints.traefik-dashbaord.address=:8080"
```

Comment:
typo

Reply:
Holy, it took a whilke to find the typo...

```
- "--entrypoints.traefik-dashbaord.address=:8080"`
                              ^
```

Thank you for your feedback.